lib/python/treadmill_aws/formatter.py

```diff
@@ -279,6 +279,8 @@ def format(item):
             ('realm', 'realm', None),
             ('instance-profile', 'instance_profile', None),
             ('subnets', 'subnets', ','.join),
+            ('s3_region', 's3_region', None),
```

Comment:
nit - should it be s3_docker_region? can it default to region of the server? or you expect to have cross region access to docker registry (can be, not saying it must be in same region necessarily).